opentech/apply/funds/management/commands/seed_community_lab_application.py

```diff
@@ -0,0 +1,101 @@
+import json
+
+from datetime import date
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from opentech.apply.funds.models import ApplicationForm, FundForm, FundType, Round
+from opentech.apply.home.models import ApplyHomePage
+from opentech.apply.users.groups import STAFF_GROUP_NAME
+
+CL_ROUND_TITLE = 'Community lab open round'
+
+
+class Command(BaseCommand):
+    help = "Pre-seeds the Community lab application form and fund type. Depends on the categories seed being run first."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        # There's an Community lab open round, so bail out. Avoids duplicate command runs.
+        if Round.objects.filter(title=CL_ROUND_TITLE).count():
+            self.stdout.write(self.style.WARNING('Skipping. The target Round/Fund Type and Application Form exist'))
+            return
+
+        application_form = self.create_community_lab_form()
+        fund = self.create_community_lab_fund_type(application_form)
+        self.create_community_lab_round(fund)
+
+    def create_community_lab_form(self):
+
+        data = [
+            {"type": "text_markup", "value": "<h3>Basic information</h3>", "id": "33d38e10-43e3-4a61-8a10-2d3ad760861b"},
+            {"type": "title", "value": {"field_label": "What is your project name?", "help_text": "", "info": None}, "id": "4507372e-a284-488a-ac3c-53b689849d67"},
+            {"type": "full_name", "value": {"field_label": "Your name", "help_text": "", "info": None}, "id": "b62f1c35-52c7-44b5-b7cd-2f1aac6701bd"},
+            {"type": "email", "value": {"field_label": "E-mail", "help_text": "", "info": None}, "id": "02a6cd0e-7ff6-46cd-9bdb-bf5858fe2514"},
+            {"type": "date", "value": {"field_label": "Date of event", "help_text": "", "required": "false", "default_value": ""}, "id": "8c74af9c-6cc7-4558-9d72-0f2c9a87f22b"},
```

Comment:
This seems to create a date field in proper order. But I get the same issue when filling in an application form manually.